### ISSUE TITLE
Option  --exclude-empty-folders doesn't work

### ISSUE THREAD

**Issue by xko** [duplicati/duplicati#1675]
In in 2.0.1.7- 2.0.1.7_experimental_2016-03-19

```
>Duplicati.CommandLine.exe test-filters C:\ --parameters-file="duplicati.params"
The supplied option --exclude-empty-folders is not supported and will be ignored

```
##

**Comment by xko** [comment]
For some reason I could not reopen #263, so creating new one

**Comment by kotor610** [comment]
has there been any update on this issue?

**Comment by warwickmm** [comment]
I don't see the warning message when I run
```
$ duplicati-cli test-filters <source path> --exclude-empty-folders
```
I suspect this was fixed in revision c6be934747a987e0a3948631a292b3cb24a1ff18.
